web/packages/teleport/src/services/storageService/storageService.ts

```diff
@@ -102,6 +102,21 @@ export const storageService = {
     return time ? time : 0;
   },
 
+  setLoginTime(time: number) {
+    const existingTime = Number(
+      window.localStorage.getItem(KeysEnum.LOGIN_TIME)
+    );
+    // Only set the login time if it doesn't already exist.
+    if (!existingTime) {
```

Comment:
What's the reason for it? Clearing the entire local storage feels wrong, we have things there that we probably don't want to clear every time we log in (like feature flags etc).

Reply:
I think you might be commenting on the wrong line? I think you meant to comment on this:

https://github.com/gravitational/teleport/blob/3dd9b395a943a60d11d98b06e1a170b62d44f0da/web/packages/teleport/src/Login/useLogin.ts#L100

I've updated this to now only clear the login time.